Library/Formula/nodenv.rb

```diff
@@ -1,19 +1,28 @@
 class Nodenv < Formula
-  desc "NodeJS version management"
-  homepage "https://github.com/wfarr/nodenv"
-  url "https://github.com/wfarr/nodenv/archive/v0.3.4.tar.gz"
-  sha256 "6d3946960beed4e69596a5f0519ad4ca7606db4e4ab1b85a3bf4add06d64290b"
-  head "https://github.com/wfarr/nodenv.git"
+  desc "Manage multiple NodeJS versions"
+  homepage "https://github.com/OiNutter/nodenv"
+  url "https://github.com/OiNutter/nodenv/archive/v0.4.0.tar.gz"
+  sha256 "bfbfb9de2177cf2418baf367b08e929459027cb8979419db7602929cb10e73bd"
+  head "https://github.com/OiNutter/nodenv.git"
 
   bottle :unneeded
 
+  depends_on "node-build" => :recommended
+
   def install
-    prefix.install "bin", "libexec"
+    inreplace "libexec/nodenv", "/usr/local", HOMEBREW_PREFIX
+    prefix.install "bin", "libexec", "completions"
   end
 
   def caveats; <<-EOS.undent
     To enable shims and autocompletion add to your profile:
       if which nodenv > /dev/null; then eval "$(nodenv init -)"; fi
+    To use Homebrew's directories rather than ~/.nodenv add to your profile:
+      export NODENV_ROOT=#{opt_prefix}
```

Comment:
Cool, thanks! ( the dangers of just migrating the file from our tap, I
guess :) )

On Mon, Nov 2, 2015 at 2:27 PM, Dominyk Tiller <notifications@github.com>
wrote:

> In Library/Formula/nodenv.rb
> <https://github.com/Homebrew/homebrew/pull/45576#discussion_r43669558>:
>
> >    end
> >
> >    def caveats; <<-EOS.undent
> >      To enable shims and autocompletion add to your profile:
> >        if which nodenv > /dev/null; then eval "$(nodenv init -)"; fi
> > +    To use Homebrew's directories rather than ~/.nodenv add to your profile:
> > +      export NODENV_ROOT=#{opt_prefix}
>
> Yes. /usr/local/opt/formula is just a symlink to
> /usr/local/Cellar/formula/1.2.3/. var is indeed the recommended place for
> things that need to be retained.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/45576/files#r43669558>.
>